claat/parser/md/parse_test.go

```diff
@@ -152,7 +154,48 @@ feedback link: https://www.google.com
 `
 	content += ("# " + title)
 
-	c := mustParseCodelab(content)
+	c := mustParseCodelab(content, parser.Options{})
+	if !reflect.DeepEqual(c.Meta, wantMeta) {
+		t.Errorf("\ngot:\n%+v\nwant:\n%+v", c.Meta, wantMeta)
+	}
+}
+
+func TestParseMetadataPassMetadata(t *testing.T) {
+	title := "Codelab Title"
+	wantMeta := types.Meta{
+		Title:      title,
+		ID:         "zyxwvut",
+		Authors:    "john smith",
+		Summary:    "abcdefghij",
+		Categories: []string{"not", "really"},
+		Tags:       []string{"kiosk", "web"},
+		Feedback:   "https://www.google.com",
+		GA:         "12345",
+		Extra: map[string]string{
```

Comment:
nit: gofmt? weird spacing. Have seen this before, but don't recall why it happens... multiline declaration?

Reply:
That is the output of gofmt.